src/image_builder.py

```diff
@@ -149,7 +149,7 @@ def image_builder(buildspec, image_types=[], device_types=[]):
 
         transformers_version = image_config.get("transformers_version")
 
-        if str(BUILDSPEC["framework"]).startswith("huggingface") or str(BUILDSPEC["framework"]).endswith("trcomp"):
+        if str(BUILDSPEC["framework"]).startswith("huggingface"):
```

Comment:
The `framework` parameter for huggingface trcomp images start with `huggingface` anyways. This condition is redundant